OrbitGl/ImGuiOrbit.cpp

```diff
@@ -728,14 +738,51 @@ void OutputWindow::Draw(const char* title, bool* p_opened, ImVec2* a_Size) {
 
   ImGui::TextUnformatted(Buf.begin());
 
-  static bool checked = true;
-  ImGui::Checkbox("blah", &checked);
+  ImGui::End();
+  ImGui::PopStyleVar();
+}
+
+namespace ImGuiOrbit {
+std::string PrettyVariableName(const char* name) {
```

Comment:
Are we sure we want such a method? If I could follow correctly so far, this is pretty printing variables, that we expose explicitly to the debug view.
This complete method looks very fragile. I'd rather suggest to either, just expose the original variable names (would be easier to find in an IDE anyways), or when exposing them, also add a description string.

Reply:
yeah I removed the function altogether...

Reply:
Thanks!